src/shared/components/case-editor/services/work-allocation.service.ts

```diff
@@ -27,15 +27,21 @@ export class WorkAllocationService {
    * Call the API to get tasks matching the search criteria.
    * @param searchRequest The search parameters that specify which tasks to match.
    */
-  public searchTasks(searchRequest: TaskSearchParameters): Observable<object> {
-    const url = `${this.appConfig.getWorkAllocationApiUrl()}/task`;
+  public searchTasks(searchRequest: TaskSearchParameter): Observable<object> {
```

Comment:
The change to this signature theoretically makes this a breaking change.

Reply:
This is used internally within the CCD-UI toolkit. Not sure why you think it will be a breaking change. Please explain. Thanks

Reply:
Because it's a `public` method. I wanted to make sure it doesn't form part of the public api for the toolkit. If it _does_, it would constitute a breaking change. If it's _entirely_ internal, it would be fine.

Reply:
Its not being used anywhere. But, Theoretically I agree. let me increment the version to major version.

Reply:
2.79.0